data/notices.json

```
@@ -1,9 +1,9 @@
 {
   "notices": [
     {
-      "title": "cli: the CDK CLI will begin to collect telemetry data on CLI usage on or after August 8, 2025.",
+      "title": "CDK CLI will collect telemetry data on command usage starting from version 2.1100.0 (unless opted out)",
```

Still too long. Changing to:

```suggestion
      "title": "CDK CLI will collect telemetry data on command usage starting at version 2.1100.0 (unless opted out)",
```